libs/domain/search/src/mocks/src/suggestion/mock-suggestion.service.ts

```diff
@@ -1,9 +1,6 @@
-import {
-  Suggestion,
-  SuggestionQualifier,
-  SuggestionService,
-} from '@spryker-oryx/search';
 import { Observable, of } from 'rxjs';
+import { Suggestion, SuggestionQualifier } from '../../../models';
```

Comment:
It's ok for mocks use alias

Reply:
Agree, but was trying if it can break. Gonna revert.